setup.py

```diff
@@ -18,7 +18,7 @@
 setup(
     name='schematic',
     version=version['__version__'],
-    description='Packages responsible for data ingress from HTAN collaborators',
+    description='Package for Data Ingress from HTAN Collaborators',
```

Comment:
Should we consider removing the reference to HTAN here? Maybe something like:

> Tools for Managing (Meta)Data Models and Validation Schemas for Data Ingress

Although, I don't know if the "for Data Ingress" qualifier is even required here.

Reply:
@milen-sage: You should probably have a say in describing your own package. 

Speaking of authorship, @sujaypatil96, there's probably a way of listing contributors alongside authors in the `setup.py` file.

Reply:
I agree if possible, we should add contributors. A note to self, we should add folks from Scripps as contributors.

I can add a description, but can worry about that later :)

Reply:
Could we change the description, even if preliminary, to:
"Package for data model and data ingress management"

Mainly not constrained to HTAN.